website/content/docs/nia/configuration.mdx

```diff
@@ -227,15 +248,62 @@ driver "terraform" {
+</EnterpriseAlert>
+
+The Terraform Cloud driver enables Consul-Terraform-Sync Enterprise to integrate with [**Terraform Enterprise**](https://www.hashicorp.com/products/terraform/editions/enterprise)<sup>1</sup>, the self-hosted distribution of Terraform Cloud. With this driver, Consul-Terraform-Sync automates Terraform runs and remote operations for workspaces.
+
+An overview of features enabled with Terraform Enterprise can be viewed within the [Network Drivers](/docs/nia/network-drivers) documentation.
+
+Only one network driver can be configured per deployment of Consul-Terraform-Sync.
+
+-> <sup>1</sup> **Upcoming**: The Terraform Cloud driver is introduced in Consul-Terraform-Sync v0.3.0 and only supports [Terraform Enterprise](https://www.hashicorp.com/products/terraform/editions/enterprise), the self-hosted distribution. The upcoming v0.4.0 release will support integration with [Terraform Cloud](https://www.terraform.io/cloud), the HashiCorp managed service.
+
+```hcl
+driver "terraform-cloud" {
+  hostname     = "my.tfe.hostname.io"
+  organization = "my-org"
+  token        = "<TEAM_TOKEN>"
+  // Optionally set the token to be securely queried from Vault instead of
+  // written directly to the configuration file.
+  // token = "{{ with secret \"secret/my/path\" }}{{ .Data.data.foo }}{{ end }}"
+
+  required_providers {
+    myprovider = {
+      source = "namespace/myprovider"
+      version = "1.3.0"
+    }
+  }
+}
+```
+
+- `hostname` - (string) The Terraform Enterprise hostname to connect to. Can be overridden with the `TFC_HOSTNAME` environment variable.
+- `organization` - (string) The Terraform Enterprise organization that hosts the managed workspaces by Consul-Terraform-Sync. Can be overridden with the `TFC_ORGANIZATION` environment variable.
+- `token` - (string) Required [Team API token](https://www.terraform.io/docs/cloud/users-teams-organizations/api-tokens.html#team-api-tokens) used for authentication with Terraform Enterprise and workspace management. Only workspace permissions are needed for Consul-Terraform-Sync. The token can also be provided using the `TFC_TOKEN` environment variable.
+  - We recommend creating a dedicated team and team API token to isolate automation by Consul-Terraform-Sync from other Terraform Enterprise operations.
+- `workspace_prefix` - (string) A prefix which is prepended to the automatically generated workspace names used for automation. This prefix will be used by all tasks which use this driver. If this prefix is not included, then the workspace will default to the task name.
+- `required_providers` - (obj: required) Declare each Terraform provider used across all tasks. This is similar to the [Terraform `terraform.required_providers`](https://www.terraform.io/docs/configuration/provider-requirements.html#requiring-providers) field to specify the source and version for each provider. Consul-Terraform-Sync will process these requirements when preparing each task that uses the provider.
```

Comment:
Thanks for the question. The Terraform `required_providers` block doesn't have any set parameters dictated by Terraform in the sense that users can configure their providers here based on their provider's configuration parameters.

Do you have any thoughts on replacing "This is similar to the Terraform `terraform.required_providers` field to specify the source and version for each provider."  => "This can be configured the same as how you would configure the Terraform `terraform.required_providers` field to specify the source and version for each provider."